docs/Configuration.md

```diff
@@ -1278,12 +1278,12 @@ export default config;
 
 #### GitHub Actions Reporter
 
-If included in the list, the built-in GitHub Actions Reporter will annotate changed files with test failure messages:
+If included in the list, the built-in GitHub Actions Reporter will annotate changed files with test failure messages and print logs with github group features for easy navigation. Note that `'default'` should not be used in this case as `'github-actions'` will handle that already, so remember to also include `'summary'`. If you wish to use it only for annotations simply pass `'silent: true'` as option (default is `false`):
 
 ```js tab
 /** @type {import('jest').Config} */
 const config = {
-  reporters: ['default', 'github-actions'],
+  reporters: [['github-actions', {silent: false}], 'summary'],
```

Comment:
Odd.. CI just ran and I was expecting to see the logs (because of `silent: false` default setting), but nothing changed:

<img width="756" alt="Screenshot 2023-01-10 at 15 17 22" src="https://user-images.githubusercontent.com/72159681/211562412-603a9b67-6b6e-47cb-8c95-9c9aca959c1d.png">

Jest’s CI configuration comes from here:

https://github.com/facebook/jest/blob/ee63afcbe7904d18558d3cc40e0940804df3deb7/jest.config.ci.mjs#L14-L15

Reply:
Now it should work